CliFx.Tests/ErrorReportingSpecs.cs

```diff
@@ -29,7 +29,10 @@ public async Task Command_may_throw_a_generic_exception_which_exits_and_prints_f
 
             // Assert
             exitCode.Should().NotBe(0);
-            stdErrData.Should().Contain("Kaput");
+            stdErrData.Should().ContainAll(
+                "System.Exception:",
+                "Kaput", "at", 
+                "CliFx.Tests");
             stdErrData.Length.Should().BeGreaterThan("Kaput".Length);
```

Comment:
You can remove this line then.

Reply:
Will remove.

Reply:
So looking at this again, this was the test that could erroneously pass even though a stack trace wasn't printed. That's because the default exception message from .NET's `Exception` class returns a message of length greater than `"Kaput".Length`. My additional check that it contains those additional strings makes sure that we're actually looking at a stack trace output and not the default exception message in the case that behavior gets reintroduced in the future.